### ISSUE TITLE
circlesoftmax loss

### ISSUE THREAD

**Issue by nanggn** [JDAI-CV/fast-reid#390]
hi! 
I want to run my code with circlesoftmax loss, but I don't know how to use it correctly. It returns pred_class_logits, should I take CE loss behind it? Or something else.
Thanks!

**Comment by L1aoXingyu** [comment]
Yes, only need a CE loss behind it. Tt's just like arcface or cosface.

**Comment by nanggn** [comment]
OK. Thanks a lot.

**Comment by nanggn** [comment]
By the way, is there need for the last fc layer for classification?

**Comment by L1aoXingyu** [comment]
You mean the last fc layer can be removed?

**Comment by SuperbTUM** [comment]
Are circle loss and cross-entropy loss mutually exclusive? Can I use both in the cls layer in training?
